src/desktop/apps/artist/server.tsx

```diff
@@ -17,21 +17,21 @@ app.get(
       const user = req.user && req.user.toJSON()
       const { ARTIST_INSIGHTS } = res.locals.sd
       const {
-        ServerApp,
+        bodyHTML,
```

Comment:
This is probably how it should have been done from the beginning. I was envisioning being able to pass props along from consuming apps (before rendering on the server) but in practice we haven't needed to do this.

Reply:
Yeah I can see how you might think that, but now we know we don’t need it for sure and can simplify 👍